buildtap.py

```diff
@@ -420,11 +420,11 @@ def inf2cat(self, x64):
     def sign(self, file):
         certspec = ""
         if self.sign_cert:
-            certspec += "/f %s " % self.sign_cert
+            certspec += "/f '%s' " % self.sign_cert
             if self.cert_pw:
-                certspec += "/p %s " % self.cert_pw
+                certspec += "/p '%s' " % self.cert_pw
         else:
-            certspec += "/s my /n %s " % self.sign_cn
+            certspec += "/s my /n '%s' " % self.sign_cn
```

Comment:
There's a bug lurking on this line. The change looked so innocent that I overlooked it, assuming it would just work.

Our certificate's _Subject_ field is _CN="OpenVPN Technologies, Inc.", O="OpenVPN Technologies, Inc.", L="Pleasanton", S="California", C="US"_. Prior to this PR this command-line produced correctly signed catalog files:

    $ python.exe buildtap.py <other-args> --cert="OpenVPN"

This translated to

    $ SignTool <other-args> /n OpenVPN <other-args>

After this PR the command-line had become

    $ SignTool <other-args> /n 'OpenVPN' <other-args>

It seems that SignTool takes the single-quotes around the certificate name to mean an exact match. Without quotes it does a substring match. So the single quotes results in a signing failure:

    SignTool Error: No certificates were found that met all the given criteria

I think the quotes make sense for _sign_cert_ and _cert_pw_, and should not break anything there. Was there a particular reason to put quotes around sign_cn? If not, then they should be removed.

Reply:
@chipitsine restoring this old thread. Yes, I agree. We could and probably should make signtool errors fatal.

Reply:
if it is not relevant, then close it.
if it is relevant (I think so), merge it

in neither case it should stay without move

Reply:
This particular PR was merged over a year ago. My fix is what's pending approval, see PR https://github.com/mattock/tap-windows6/commit/65b29e33178572b1dc76174e94de6bf10624b473.